test/jest/rendering-utils.js

```diff
@@ -1,6 +1,5 @@
 import { render } from "@testing-library/react";
-import * as React from "react";
 
 export const renderInSvg = (component) => {
-  return render(<svg>{component}</svg>);
+  return render(component, { wrapper: "svg" });
 };
```

Comment:
I had forgotten that there is a slightly easier way to add a custom wrapper component. Does it still make sense to have this helper, or is it just as clear if we just add `{ wrapper: "svg" }` to each individual test where we need it, rather than having to remember to use this helper?

Reply:
I think I will just remove this so we don't have to remember another import. FYI @scottrippey